spec/views/users/passwords/edit.html.erb_spec.rb

```diff
@@ -35,4 +41,20 @@
       ),
     )
   end
+
+  context 'required password change' do
+    let(:required_password_change) { true }
```

Comment:
Can we assert that it doesn't have a "Cancel" link? And assert above in the normal case that it does?